lib/cluster/index.ts

```diff
@@ -348,8 +348,12 @@ class Cluster extends EventEmitter {
       const node = nodes[index]
       debug('getting slot cache from %s:%s', node.options.host, node.options.port)
       _this.getInfoFromNode(node, function (err) {
-        if (_this.status === 'end') {
-          return wrapper(new Error('Cluster is disconnected.'))
+        switch (_this.status) {
+          case 'close':
+          case 'end':
```

Comment:
Same error returned for 3 statuses, was this intentional?

Reply:
"disconnecting" returns a different error.

Reply:
ah, didn't notice the difference at the end